src/main/java/org/gitlab/api/models/GitlabUser.java

```diff
@@ -22,7 +22,7 @@
     private String _twitter;
     private String _provider;
     private String _state;
-    private boolean _blocked;
+    private Boolean _blocked;
```

Comment:
This change causes an NPE during json serialization when `_blocked` is `null` and `isBlocked()` attempts to auto-unbox.

Reply:
I guess isBlocked() should also return a Boolean.

Reply:
https://github.com/timols/java-gitlab-api/pull/256